masu/external/downloader/aws/aws_report_downloader.py

```diff
@@ -194,5 +193,8 @@ def download_report(self, date_time):
             stats_recorder.update(etag=etag)
             stats_recorder.commit()
 
-            files.append(file_name)
-        return files
+            report_dictionary['file'] = file_name
+            report_dictionary['compression'] = self.report.get('Compression')
```

Comment:
This is more a fault on my end, because I think my solution was too rigid. Right now if compression is not either `''` or `'gzip` the processor will fail. I think I should have allowed `None` to mean no uncompressed as well. I can turn around with a quick fix for that.

Reply:
OK sounds good

Reply:
You are right. For some reason I though uncompressed was an option, but they will always compress the files.